lib/Command/ListCommand.php

```diff
@@ -87,8 +87,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
 		if ($outputType === self::OUTPUT_FORMAT_JSON || $outputType === self::OUTPUT_FORMAT_JSON_PRETTY) {
 			foreach ($folders as &$folder) {
-				$folder['group_details'] = $folder['groups'];
```

Comment:
Why do you remove group_details? This is a breaking change for any script relying on that.

Reply:
Yes, as I said at the top.
This was discussed in last weeks team call with @sorbaugh.
It is fine to do this once for the benefit of a much cleaner API.